src/Framework/Constraint/IsResource.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PHPUnit\Framework\Constraint;
+
+/**
+ * Constraint that checks if value is resource
+ *
+ * The file path to check is passed as $other in evaluate().
+ */
+class IsResource extends Constraint
```

Comment:
`IsResource` is not needed as a `Constraint` object. `IsType('resource')` already exists. What I meant in https://github.com/sebastianbergmann/phpunit/pull/2808/commits/72aae12e7326ef0ae86bc821bfb9f5ce9a47832b#r144721495 was a `Constraint` object that checks a specific resource type.

Reply:
Now I understand, thanks for review.